guide/daily-spending-wallet/requesting.md

```diff
@@ -34,440 +34,400 @@ Illustration source
+   alt-text = "Screen showing a toggle between lightning and on-chain request."
+   width = 250
+   height = 541
+   modalWidth = 250
+   modalHeight = 541
+%}
 
 </div>
 
-### Metadata
-
-A payment request with only standard data, like an amount and date, communicates little to both parties about purpose and context of the payment.
+## Metadata
 
 <div class="center" markdown="1">
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings@2x.png"
+   image = "/assets/images/guide/daily-spending-wallet/requesting/Fees@2x.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/Fees@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/Fees@2x.png"
    layout = "float-left-desktop -background -shadow"
-   caption = "Let users attach payment metadata to single-use payment requests."
-   alt-text = "Screen showing single payment request settings."
+   caption = "Let users know if fees will be charged."
+   alt-text = "Screen showing a payment request with setup fees shown."
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
-Users should have the option to attach a note for record-keeping and for the sender to read, tags, labels and any other metadata that may be relevant to give more context to the payment.
+Metadata is additional information that is part of or can be added to a payment request. This can include an amount, note, or name for the sender. Or for the requester, labels, or [assigning a contact]({{ "/guide/daily-spending-wallet/contacts/#adding-a-contact-to-an-outgoing-invoice" | relative_url }}).
 
-This metadata should also be able to be [backed up]({{ "/guide/onboarding/backing-up-a-wallet/" | relative_url }}) in case the user loses access to their wallet and has to restore it.
+Expiration times are something unique to lightning invoices. We cover modifying these [here]({{ "/guide/daily-spending-wallet/requesting/#invoice-expirations" | relative_url }}).
 
-#### Add a requester name with NameDesc
-
-You can convey the name of the user requesting the payment. Suppose the user is able to define their name in the settings of their wallet app. The app could then append the user's name to the beginning of the description field in a BOLT11 invoice. For example, if Alice makes an invoice with a memo that says "For design work", then the description in the BOLT11 invoice would become:
-
-```
-Alice:  For design work
-```
-
-This description is human-readable, and wallets that support [bLIP-0011 NameDesc](https://github.com/lightning/blips/blob/master/blip-0011.md) will parse this as a name and a description.
+Ensure users can [backup]({{ "/guide/daily-spending-wallet/backup-and-recovery/landing-page/" | relative_url }}) this metadata to prevent them from losing their transaction history.
```

Comment:
```suggestion
Ensure users can [back up]({{ "/guide/daily-spending-wallet/backup-and-recovery/landing-page/" | relative_url }}) this metadata to prevent them from losing their transaction history.
```

Used as a verb in this context